src/test/java/com/softserveinc/edu/ita/pageobjects/PageObjectBase.java

```diff
@@ -54,6 +56,11 @@ public void sendKeys(ILocator locator, String text) {
         Reporter.log(String.format("<br>INFO&nbsp;&nbsp; - Typed '%s' in <b>%s</b>", text, locator.getName()));
     }
 
+    public void clear(ILocator locator){
```

Comment:
this is not a generic method, you are clearing the search field on the Ordering page, move it out of base